src/components/ec-btn-dropdown/ec-btn-dropdown.spec.js

```diff
@@ -115,7 +136,7 @@ describe('EcBtnDropdown', () => {
         },
         {
           attrs: {},
-          href: '/payments/?add-new-payment',
+          href: '#/payments/',
```

Comment:
Just curiosity, why `href` is changed?

Reply:
I was getting errors from JSDOM that navigation is not implemented only hash navigation is supported. We have tests where we click on anchors with this href - to remove the errors, I changed the hrefs to hashes.